pkg/virt-config/config-map.go

```diff
@@ -360,6 +362,20 @@ func setConfigFromConfigMap(config *v1.KubeVirtConfiguration, configMap *k8sv1.C
 		config.DeveloperConfiguration.FeatureGates = stringToStringArray(featureGates)
 	}
 
+	config.ObsoleteCPUs = make(map[string]bool)
+	if val, ok := configMap.Data[ObsoleteCPUsKey]; ok {
+		if cpuModels := strings.Replace(strings.TrimSpace(val), " ", "", -1); cpuModels != "" {
+			cpuModels := strings.Split(cpuModels, ",")
+			for _, v := range cpuModels {
+				config.ObsoleteCPUs[v] = true
+			}
+		}
+	}
```

Comment:
I think we should make this list of obsolete cpu models be configurable on kubevirt CR. The config map will go away soon.
You can probably add these two structs (obsolete and minimum CPU) somewhere here : 
https://github.com/kubevirt/kubevirt/blob/master/staging/src/kubevirt.io/client-go/api/v1/types.go#L1559